tests/lib/Repository/Values/Content/DefaultLanguageCodeForContentTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Core\Repository\Values\Content;

```

Comment:
If you look at `ibexa/core` directory/namespace structure, this namespace is reserved for unit tests. AFAICS it should be rather placed in the new directory `./tests/integration/Core/Repository/Values/Content` as:
```suggestion
namespace Ibexa\Tests\Integration\Core\Repository\Values\Content;
```

Reply:
Ok - on it